src/pages/core/create-asset.md

```diff
@@ -230,16 +230,32 @@ MPL Core Assets can be created straight into a collection if your MPL Core Colle
 
 ```ts
 import { generateSigner } from '@metaplex-foundation/umi'
-import { createCollection, create } from '@metaplex-foundation/mpl-core'
+import {
+  createCollection,
+  create,
+  fetchCollection,
+} from '@metaplex-foundation/mpl-core'
 
 const collectionSigner = generateSigner(umi)
 
+// create collection
+// if you are doing this in a single script you may have
+// to use a sleep function or commitment level of 'finalized'
+// so the collection is fully written to change before fetching it.
 await createCollection(umi, {
   collection: collectionSigner,
   name: 'My Collection',
   uri: 'https://example.com/my-collection.json',
 }).sendAndConfirm(umi)
 
+// fetch the collection
+// if the createCollection transaction succeeds but
+// the fetch produces 'accountNotFound' then you will need
+// to sleep before executing the fetch.
```

Comment:
Or use finalized right?  This comment could be here but if it is meant to be redundant to the first one might be clearer to just remove it, or put the first comment over the fetch?

Reply:
yeah you are right. Fixed.